source/stylesheets/application.scss

```diff
@@ -1,17 +1,66 @@
+//= require_directory ./components
 @charset "utf-8";
 @import "normalize";
 @import "bourbon";
+@import "settings/grid_layout";
 @import "neat";
 @import "mixins/fonts";
 @import "settings/colors";
 
+$base-font-size: 14px;
+
 body {
   @include font-opensans;
-  font-size: 16px;
-  line-height: 21px;
+  font-size: $base-font-size;
   color: $text-color;
 }
 
-#application {
+.page-header{
+  background: $header-background;
+  color: $header-color;
+  text-shadow: 1px 1px 3px rgba(0,0,0,0.5);
+  @include box-sizing(border-box);
+
+  a {
+    color: inherit;
+  }
+}
+
+.page-header-container {
```

Comment:
if you change the classes like mentioned above you can simply open up a single `.page-header {` block and select the nested stuff like so:

```scss
.page-header {
 // styles …
 .container {
   // styles …
  }

  .content {
    // styles …
  }
}